src/playback/config.js

```diff
@@ -123,6 +123,7 @@ const PLATFORMS = [
 
 const BROWSERS =[
   {id: "firefox", label:"Firefox", title:"Firefox (Desktop)", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-firefox-live"]}}},
+  {id: "chrome", label:"Chrome", title:"Chrome (Desktop)", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-chrome-live"]}}},
```

Comment:
...yes:  We should be adding a property to these records, like `{format: "desktop"}`, and using that.